app/src/main/java/app/gamenative/ui/component/dialog/ElementEditorDialog.kt

```
@@ -431,6 +567,254 @@ fun ElementEditorDialog(
+                        ) {
+                            Slider(
+                                value = currentLookSensitivity,
+                                onValueChange = {
+                                    currentLookSensitivity = it
+                                    hasUnsavedChanges = true
+                                },
+                                valueRange = 0.1f..10.0f,
+                                modifier = Modifier.weight(1f)
+                            )
+                            Text(
+                                text = String.format("%.1fx", currentLookSensitivity),
```

detekt (1.23.8)

[warning] String.format("%.1fx", currentLookSensitivity) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)